vault/blog.2021.2021-12-30-five-minute-journal-template.md

```diff
@@ -0,0 +1,151 @@
+---
+id: P1DL2uXHpKUCa7hLiFbFA
+title: 'Grow with Gratitude and Mindfulness: Using The Five Minute Journal template in Dendron'
+desc: >-
+  Start your day, and end your day, with a simple journal method. The Five
+  Minute Journal (5MJ) helps you express gratitude, make your days better, and
+  focus on mindfulness.
+updated: 1640814353154
+created: 1640790736919
+date: '2021-10-25'
+excerpt: >-
+  Start your day, and end your day, with a simple journal method. The Five
+  Minute Journal (5MJ) helps you express gratitude, make your days better, and
+  focus on mindfulness.
+author: Derek Ardolf
+image:
+  url: #todo
+  alt: #todo
+publish: false
+---
+
+> Banner image #todo
+
+[Dendron](https://dendron.so) helps people and products organize with notes, docs, and knowledge of all the things. A way to make life easier is to use templates. But where to start?
+
+## The Five Minute Journal
+
+> Add image of journal #todo
+> - Five Minute Journal images: https://drive.google.com/drive/folders/1D8KtyXDvsyL7BTqmZ336Lukzxwx1jeDr
+
+One of the most common ways of note taking is in the form of a journal. Let's make it easier for daily use with a template based off of a popular journaling format: [_The Five Minute Journal_](https://www.intelligentchange.com/pages/customers#journal), by [Intelligent Change](https://www.intelligentchange.com/pages/our-story).
+
+The Five Minute Journal (5MJ) helps you express gratitude, make your days better, and focus on mindfulness. For an example of journaling with this method, Tim Ferriss uploaded a video in 2020 all about his approach with _physical notebooks_: [How I Journal and Take Notes: Brainstorming + Focusing + Reducing Anxiety](https://youtu.be/UFdR8w_R1HA).
```

Comment:
provide context.
Who is Tim Ferriss and why do we care? (eg. Tim Ferris, world famous entrepreneur, investor and lifestyle guru)